### PR TITLE
python-Flask: update to 1.0.1

### PR DESCRIPTION
**Pull request by pulux**

### CODE REVIEW

File: srcpkgs/python-Flask/template [hunk 1]
```diff
@@ -13,7 +13,7 @@ maintainer="Markus Berger <pulux@pf4sh.de>"
 homepage="http://flask.pocoo.org"
 license="3-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/python-Flask/template]: BSD-3-Clause, move it above homepage.

File: srcpkgs/python-Flask/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'python-Flask'
 pkgname=python-Flask
-version=0.12.2
-revision=2
+version=1.0.1
```
Comment (cr6git) [inline srcpkgs/python-Flask/template]: Latest version is 1.0.2

**Comment by pulux** [comment]
I think, we should wait for a stable version.

**Comment by pulux** [comment]
for older software with maybe breaks with 1.0.x we should use first #14129 and version 1.0.x has many releases in the last 8 days.

**Pull request closed without merging**